molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -180,6 +180,9 @@
 <div class="row">
 <div class="col-md-12">
 <div id="plugin-container">
+    <#if plugin_description??>
+        ${plugin_description}
```

Comment:
Inserting raw HTML is dangerous and could lead to vulnerabilities. I suggest using TEXT type and for Plugin.DESCRIPTION and render all descriptions as ${plugin_description?html} with a similar style regardless of content